include/domain-types.h

```diff
@@ -215,14 +215,16 @@ struct ObjectStream
 
 struct Count
 {
-    Count(const timestamp_t& timestamp, int32_t value)
+    Count(const timestamp_t& timestamp, std::string& label, int32_t value)
         : timestamp(timestamp)
+        , label(label)
         , value(value)
     {
     }
 
     timestamp_t timestamp;
-    int32_t value;
+    std::string label;
```

Comment:
Please, move it to the end of the list. I know, in JSON fields are in this order, but when extending structure new items usually put at the end, not in the middle or start, unless there are reasons not to do it (I see no reasons in this case)